README.md

```diff
@@ -1,68 +1,79 @@
 # 1Password SCIM Bridge deployment examples
 
-You can deploy 1Password SCIM Bridge on any supported infrastructure that allows ingress by your identity provider and egress to 1Password servers. Here you'll find configuration files and best practices to help with your deployment.
+You can deploy 1Password SCIM Bridge on any supported infrastructure that allows ingress from your identity provider and egress to 1Password servers. Here you'll find configuration files and best practices to help with your deployment.
 
-> [!TIP]
-> Before you deploy your SCIM bridge, learn more about [automating provisioning in 1Password using SCIM](https://support.1password.com/scim/).
+## Before you begin
 
-## Marketplace apps
+Before you begin deploying 1Password SCIM Bridge,
 
-For an automated deployment of 1Password SCIM Bridge where some settings are pre-configured for you, use a marketplace app from Google Cloud Platform or DigitalOcean:
+- learn more about [automating provisioning in 1Password using SCIM](https://support.1password.com/scim/).
+- review the [Preparation Guide](/PREPARATION.md). The guide will help you plan for some of the technical components of the deployment and consider some issues you may encounter along the way.
 
-- [Deploy 1Password SCIM Bridge on Google Cloud Platform](https://support.1password.com/scim-deploy-gcp/)
-- [Deploy 1Password SCIM Bridge on DigitalOcean](https://support.1password.com/scim-deploy-digitalocean/)
+## Cloud platforms examples
 
-## Custom deployment
+Platform-native solutions sorted by cloud service provider:
```

Comment:
Nitpick: Consider simplifying the subheading to 'Solutions by cloud platform' for improved brevity and readability, echoing terminology in main heading.
```suggestion
Solutions by cloud platform:
```